cli-subcommands/create-container.js

```diff
@@ -103,7 +101,7 @@ exports.run = function(probo) {
 
 function exitWithError(message) {
   console.error(message);
-  process.exit(1);
```

Comment:
You can actually suppress this specific check by adding `/*eslint no-process-exit: 0*/` at the top of the file.